### PR TITLE
chore(datastore): Update changelog with preview instructions

### PR DESCRIPTION
**Pull request by HuiSF**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_datastore/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
+## 0.3.0-rc.2 (2021-10-25)
+
+This release candidate requires to install `@aws-amplify/cli@6.2.0-custom-type-preview.0`
```
Comment (fjnoyp) [inline packages/amplify_datastore/CHANGELOG.md:3]: I think the wording is a bit confusing here.  Could we say something like: This release candidate requires a preview version of amplify-cli that can be downloaded with this command: ""

File: packages/amplify_datastore/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+
+```
+npm uninstall -g @aws-amplify/cli && npm install -g @aws-amplify/cli@6.2.0-custom-type-preview.0
+```
+and to regenerate models.
```
Comment (fjnoyp) [inline packages/amplify_datastore/CHANGELOG.md:8]: and to regenerate models by running the following command:

**Review by fjnoyp — APPROVED** [review]
Minor rephrasing comments to make things clearer for end users.  Otherwise looks good.

**Review by dnys1 — APPROVED** [review]
Agree with Kyle's comments, otherwise LGTM

**Pull request merged**